atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/pue/CassandraImitatingConsensusForgettingStoreTest.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * (c) Copyright 2021 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.pue;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import com.palantir.atlasdb.encoding.PtBytes;
+import com.palantir.atlasdb.keyvalue.api.Cell;
+import com.palantir.atlasdb.keyvalue.api.CheckAndSetException;
+import com.palantir.atlasdb.keyvalue.api.KeyAlreadyExistsException;
+import java.util.Optional;
+import java.util.concurrent.ExecutionException;
+import org.junit.Test;
+
+public class CassandraImitatingConsensusForgettingStoreTest {
+    private static final Cell CELL = Cell.create(new byte[] {1}, new byte[] {0});
+    private static final byte[] VALUE = PtBytes.toBytes("VAL");
+    private static final byte[] VALUE_2 = PtBytes.toBytes("VAL2");
+    private static final double PROBABILITY_THROWING_ON_QUORUM_HALF = 0.16;
```

Comment:
Is this the solution to the equation `(1-x)^4 = 0.5`, and if so could we just give either
- a comment and a bit more precision, or
- the mathematical expression?